docker/filebeat/filebeat.yml

```diff
@@ -25,190 +25,142 @@ filebeat.autodiscover:
       templates:
         - condition:
             contains:
-              docker.container.image: "apm-server"
+              docker.container.name: "opbeans-"
           config:
             - type: container
               paths:
-                - '/var/lib/docker/containers/*/*.log'
-              containers.ids:
-                - "${data.docker.container.id}"
+                - "/var/lib/docker/containers/*/${data.docker.container.id}-json.log"
+              include_lines: ['^{']
+              tail_files: true
+              processors:
+              - add_tags:
+                  tags: [json]
+                  target: "parser_type"
+              - decode_json_fields:
+                  fields:
+                    - message
+                  target: "json"
+                  overwrite_keys: true
+                  add_error_key: true
               fields_under_root: true
-              json.keys_under_root: true
-              json.overwrite_keys: true
-              json.add_error_key: true
-              json.message_key: message
         - condition:
             contains:
-              docker.container.image: "filebeat"
+              docker.container.name: "opbeans-"
```

Comment:
Same condition twice?

Reply:
yep, this is only for the no JSON log lines, I know that we can process it in the same config, but for the moment to make both config as simple as possible I've split the config in two, JSON and NO_JSON